extensions-core/lookups-cached-global/src/main/java/org/apache/druid/query/lookup/namespace/ExtractionNamespace.java

```diff
@@ -49,4 +49,9 @@ default long getJitterMills()
   {
     return 0;
   }
+
+  default long getLoadTimeoutMills()
+  {
+    return 60 * 1000;
```

Comment:
QQ. Is there any reason we have kept this to be 60 sec but the one for default on Jdbc extraction namespace is 120 sec ?

Reply:
yeah, default loadTime for lookups is 60 secs that will be enough time to load all non jdbc lookups like uri, maps, and other loaders. Overriding this loadTime for JDBC to 2 mins as it can take little longer to load entries if they in millions. Also documented it so that users can customize it for their use cases of big lookups.